packages/react-native-sdk/src/theme/theme.ts

```diff
@@ -0,0 +1,388 @@
+    participantStatusContainer: ViewStyle;
+    avatarContainer: ViewStyle;
+    userNameLabel: TextStyle;
+    audioMutedIconContainer: ViewStyle;
+  };
+  participantLabel: {
+    container: ViewStyle;
+    userNameLabel: TextStyle;
+    audioMutedIconContainer: ViewStyle;
+    videoMutedIconContainer: ViewStyle;
+    pinIconContainer: ViewStyle;
+    screenShareIconContainer: ViewStyle;
+  };
+  participantNetworkQualityIndicator: {
+    container: ViewStyle;
+  };
+  participantReaction: {
+    container: ViewStyle;
+    reaction: ViewStyle;
+  };
+  participantVideoFallback: {
+    container: ViewStyle;
+    label: TextStyle;
+  };
+  participantView: {
+    container: ViewStyle;
+    footerContainer: ViewStyle;
+    highligtedContainer: ViewStyle;
+  };
+  videoRenderer: {
+    container: ViewStyle;
+    videoStream: ViewStyle;
+  };
+  incomingCall: {
+    background: ViewStyle;
+    content: ViewStyle;
+    incomingCallText: TextStyle;
+    buttonGroup: ViewStyle;
+  };
+  outgoingCall: {
+    container: ViewStyle;
+    background: ViewStyle;
+    callingText: TextStyle;
+    buttonGroup: ViewStyle;
+    deviceControlButtons: ViewStyle;
+  };
+  callTopView: {
+    container: ViewStyle;
+    content: ViewStyle;
+    backIconContainer: ViewStyle;
+    leftElement: ViewStyle;
+    centerElement: ViewStyle;
+    rightElement: ViewStyle;
+    title: TextStyle;
+  };
+  userInfo: {
+    container: ViewStyle;
+    avatarGroup: ViewStyle;
+    name: TextStyle;
+  };
+  reactionsPicker: {
+    reactionsPopup: ViewStyle;
+    reactionsButtonDimmer: ViewStyle;
+    reactionItem: ViewStyle;
+    reactionText: TextStyle;
+  };
+  lobbyControls: {
+    container: ViewStyle;
+  };
+  participantInfoBadge: {
+    container: ViewStyle;
+    participantCountContainer: ViewStyle;
+    participantsIconContainer: ViewStyle;
+    participantsCountText: TextStyle;
+  };
+};
+
+export const defaultTheme: Theme = {
+  variants: {
+    buttonSizes: {
+      xs: 40,
+      sm: 50,
+      md: 60,
+      lg: 70,
+      xl: 80,
+    },
+    iconSizes: {
+      xs: 15,
+      sm: 20,
+      md: 25,
+      lg: 30,
+      xl: 35,
+    },
+    avatarSizes: {
+      xs: 50,
+      sm: 100,
+      md: 120,
+      lg: 180,
+      xl: 80,
+    },
+  },
+  typefaces: {
+    heading4: {
+      fontSize: 34,
+      fontWeight: '400',
+    },
+    heading5: {
+      fontSize: 24,
+      fontWeight: '400',
+    },
+    heading6: {
+      fontSize: 20,
+      fontWeight: '500',
+    },
+    subtitle: {
+      fontSize: 16,
+      fontWeight: '400',
+    },
+    subtitleBold: {
+      fontSize: 16,
+      fontWeight: '500',
+    },
+    bodyBold: {
+      fontSize: 16,
+      fontWeight: '600',
+    },
+    caption: {
+      fontSize: 10,
+      fontWeight: '400',
+    },
+  },
+  colors: colors,
+  avatar: {
+    container: {},
+    image: {},
+    text: {},
+  },
+  acceptCallButton: {
+    container: {},
+    svgContainer: {},
+  },
+  callContent: {
+    container: {},
```

Comment:
Seems like it is not applied to the CallContent component. You might have missed it.